katalog/etcd-backup-pvc/kustomization.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kube-system
+
+resources:
+  - cronjob.yaml
+
+images:
+  - name: etcd-backupper
+    newName: registry.sighup.io/fury/dr/etcd-backup/etcd-backupper
+    newTag: alpine3.21-v3.5.19
+  - name: rclone
+    newName: registry.sighup.io/fury/dr/etcd-backup/rclone
+    newTag: v1.69-stable
+
+patches:
+  - patch: |-
+      apiVersion: batch/v1
+      kind: CronJob
+      metadata:
+        name: etcd-backup-pvc
+      spec:
+        schedule: "* * * * *"
+  - patch: |-
+      - op: replace
+        path: /spec/jobTemplate/spec/template/spec/volumes/2/persistentVolumeClaim/claimName
+        value: test
+    target:
+      group: batch
+      version: v1
+      kind: CronJob
+      name: etcd-backup-pvc
```

Comment:
What's the idea behind this patch here? Is it for testing purposes? Let's move it to /katalog/tests instead.

If you leave this patch in the main kustomization.yaml file it will end up in the final build of the package, so if I import the package from somewhere else like KFD, I'll end up with this patch applied.

Reply:
The idea behind this is to offer people using this as a blueprint for their custom patch. I don't know, maybe it's better to ditch this :)

Reply:
If it's just as an example, you can put it in the readme and call it a day